test/secmemtest.c

```diff
@@ -10,13 +10,16 @@
 #include <openssl/crypto.h>
 
 #include "testutil.h"
+#include "../e_os.h"
```

Comment:
I'm a bit confused here , because secmemtest.c needs to `#include "../e_os.h"` while the other tests `#include "e_os.h"`:

```
msp@msppc:~/src/openssl$ grep -rnH 'include.*e_os.h"' test/
test/secmemtest.c:13:#include "../e_os.h"
test/ssltestlib.c:15:#include "e_os.h"
test/ssltest_old.c:12:#include "e_os.h"
```
The include statement in ssltestlib.c seems to be redundant (the code compiles if it is removed), but the include statement in ssltest_old.c is not redundant. I didn't see a difference in the compiler command line, so how does ssltest_old.c find its include?